StripeFinancialConnections/StripeFinancialConnections/Source/Native/NativeFlowController.swift

```diff
@@ -881,6 +881,9 @@ extension NativeFlowController: AttachLinkedPaymentAccountViewControllerDelegate
 // MARK: - NetworkingLinkVerificationViewControllerDelegate
 
 extension NativeFlowController: NetworkingLinkVerificationViewControllerDelegate {
+    func networkingLinkVerificationViewController(_ viewController: NetworkingLinkVerificationViewController, receivedConsumerPublishableKey consumerPublishableKey: String) {
```

Comment:
Nit: I guess this would need to be `didReceiveConsumerPublishableKey` to be consistent.

Reply:
Updated - good call out!